src/cli.rs

```diff
@@ -37,7 +37,7 @@ pub enum SweepCommand {
 )]
 pub struct Args {
     /// Path to check
-    pub path: Option<PathBuf>,
+    pub path: Option<Vec<PathBuf>>,
```

Comment:
Can this just be `Vec<PathBuf>` instead?

Reply:
Having it as an option makes defaulting to the current invocation path easier imo, happy to change it though!

Reply:
Easy to change it to this if you'd prefer
```rust
    let paths = match args.path.len() {
        0 => vec![env::current_dir().expect("Failed to get current directory")],
        _ => args.path,
    };
```

Reply:
When is it possible to have `Some(vec![])` here? If the answer is "never" then I prefer the `match` you just posted, yeah.

Reply:
Added